src/Http/Response/ApiProblemJsonResponse.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Response;
+
+use Crell\ApiProblem\ApiProblem;
+use Zend\Diactoros\Response\JsonResponse;
+
+/**
+ * Response type application/problem+json.
+ *
+ * @see https://tools.ietf.org/html/draft-nottingham-http-problem-07
+ **/
+class ApiProblemJsonResponse extends JsonResponse
+{
+    public function __construct(ApiProblem $problem, $headers = [])
+    {
+        $headers += ['Content-Type' => 'application/problem+json'];
+
+        $status = 400;
+        if (null !== $problem->getStatus()) {
+            $headers += [
+                'X-Status-Code' => $problem->getStatus(),
```

Comment:
I think `X-Status-Code` is not needed anymore. It seems to be something related to Symfony: https://symfony.com/blog/new-in-symfony-3-3-deprecated-x-status-code